docs/messaging/messaging-spans.md

```diff
@@ -193,32 +192,28 @@ The following operation types related to messages are defined for these semantic
 | `create`       | A message is created or passed to a client library for publishing. "Create" spans always refer to a single message and are used to provide a unique creation context for messages in batch publishing scenarios. |
 | `publish`      | One or more messages are provided for publishing to an intermediary. If a single message is published, the context of the "Publish" span can be used as the creation context and no "Create" span needs to be created. |
 | `receive`      | One or more messages are requested by a consumer. This operation refers to pull-based scenarios, where consumers explicitly call methods of messaging SDKs to receive messages. |
-| `process`      | One or more messages are delivered to or processed by a consumer. |
+| `process`      | One or more messages are processed by a consumer. |
 | `settle`       | One or more messages are settled. |
 
 ### Span kind
 
-[Span kinds](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.33.0/specification/trace/api.md#spankind)
-SHOULD be set according to the following table, based on the operation type a span describes.
+Span kind SHOULD be set according to the following table, based on the operation type a span describes.
 
 | Operation type | Span kind|
 |----------------|-------------|
 | `create`       | `PRODUCER` |
-| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context. |
-| `receive`      | `CONSUMER` |
-| `process`      | `CONSUMER` for push-based scenarios where no `receive` span exists. |
-
-For cases not covered by the table above, the span kind should be set according
-to the [generic specification about span kinds](https://github.com/open-telemetry/opentelemetry-specification/tree/v1.33.0/specification/trace/api.md#spankind),
-e. g. it should be set to CLIENT for the "Publish" span if its context is not
-used as creation context and if the "Publish" span models a synchronous call to
-the intermediary.
+| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context, `CLIENT` otherwise. |
+| `receive`    | `CONSUMER` |
+| `process`    | `CONSUMER` for push-based scenarios where no "Receive" span exists, `SERVER` otherwise. |
```

Comment:
```suggestion
| `publish`      | `PRODUCER` if the context of the "Publish" span is used as creation context. |
| `receive`    | `CONSUMER` |
| `process`    | `CONSUMER` for push-based scenarios where no "Receive" span exists. |
```

Could we leave this out of this PR, and discuss this in a separate issue/PR?

I know of some tools (including Grafana Application Observability) that assume ephemeral (uninstrumented) services for client spans without corresponding server spans (and vice versa). This assumption can be valid for the "publish" case (where a possible server span would be part of a broker trace), however, it's not valid for the "process" case (there isn't really any upstream client span).

I'm not in principal against this change, but I'd like to think and reason more about it independently of metrics.

Reply:
I created https://github.com/open-telemetry/semantic-conventions/issues/1112 to discuss this further.

Reply:
make sense, removed